getting-started.md

```diff
@@ -58,7 +58,7 @@ For more information on using the OpenFaaS and AWS EventBridge Processor, please
 
 *zAdvanced (Optional)*
   * Debugging - When enabled, this will output a more verbose log file that can be used to troubleshoot failed deployments
-  * POD CIDR Network - Customize POD CIDR Network (Default 10.99.0.0/20)
+  * POD CIDR Network - Customize POD CIDR Network (Default 10.99.0.0/20). This subnet must not overlap with the vCenter Event Broker IP address.
```

Comment:
Should this be in this PR?

Reply:
This is the reason I couldn't deploy the appliance into VMC before we could customize the networks. If you don't know Kubernetes, I think you could easily ignore this POD CIDR. I thought about putting it in troubleshooting, but why let them make the mistake when you can prevent it in the getting started guide?